### DISCUSSION TITLE
Why I received alert in my case using flatline

### DISCUSSION THREAD

**Discussion by SamMc75** [jertel/elastalert2#959] [Q&A] [answered]
Hi, I want to create a rule for alert when count of docs for filter above 0. 

I try to use flatline. 
I see in Kibana that where is no docs for filter for my query but I receive alert anyway and in log of elastalert I see this:
INFO:elastalert:Ran Flat Check errors from 2022-09-21 18:15 to 2022-09-21 18:25: 0 query hits (0 already seen), 2 matches, 1 alerts sent. 

There is my fault?
This is my rule:

index: app-log-*

buffer_time:
  minutes: 10

run_every:
  minutes: 10

name: Flat Check errors

type: flatline

threshold: 1

timeframe:
  minutes: 10

filter:
- query:
    query_string:
      query: 'level:ERROR -message:postCreateEvents*'

alert:
- "telegram"

telegram_bot_token: "hide_token"
telegram_room_id: "hide_id"

alert_text: "\n Count errors: {0}. \n\n Message: {1}. \n\n Stack_trace: {3}. \n\n"
alert_text_type: alert_text_only
alert_text_args:
    - num_hits
    - message
    - stack_trace

**Comment by jertel** [comment, marked as answer]
Flatline rules alert when there were records previously but now there are no longer records present. From your description above, it sounds like this _flatline_ rule is working as designed. Perhaps you are looking for a _frequency_ rule instead. Also, please review #11 before posting discussions on this forum.
